code/modules/unit_tests/_unit_tests.dm

```diff
@@ -53,7 +53,10 @@
 
 #define TEST_PRE 0
 #define TEST_DEFAULT 1
-#define TEST_DEL_WORLD INFINITY
+/// This should be one of the last tests to run
+#define TEST_MONKEY_BUSINESS 10
+/// This must be the last test to run
+#define TEST_CREATE_AND_DESTROY INFINITY
```

Comment:
```suggestion
/// This should be one of the last tests to run, because we run it for about 30 seconds by design and we want to run all of the tests that take a trivial amount of time to complete beforehand.
#define TEST_MONKEY_BUSINESS 10
/// This must be the last test to run due to the inherent nature of the test iterating every single tangible atom in the game, as well as the fact that it takes a hefty amount of time to run.
#define TEST_CREATE_AND_DESTROY INFINITY
```

I think it's probably a good idea to explain _why_ we're assigning these funny numbers to these test priorities so that whichever poor sap comes along in the future to add more funny-number-defines can figure out why we did it like this.

Let me know if my logick is completely incorrect, I'm just assuming that we've assigned priorities like this in the past for these reasons.